src/main/java/apoc/load/Jdbc.java

```diff
@@ -221,11 +226,23 @@ public boolean hasNext() {
         }
 
         private Object convert(Object value) {
+
             if (value instanceof UUID || value instanceof BigInteger || value instanceof BigDecimal) {
                 return value.toString();
             }
             if (value instanceof java.util.Date) {
```

Comment:
shouldn't this also take the format of the SQL datatype into account?
java.sql.Timestamp  vs. java.sql.Date vs. ... ?

Reply:
Hi @jexp the java.sql date types all inherits from `java.util.Date`